### DISCUSSION TITLE
Use with Documentation Tool

### DISCUSSION THREAD

**Discussion by oraqlle** [vector-of-bool/dds#132] [Q&A]
Is there anyway to use DDS with a documentation tool so that it builds/tests a package but also builds the documentation? Regards

**Comment by vector-of-bool** [comment]
At the moment, no. This would be outside of the project's current scope, but may be worth investigating in the future. Currently, it would be best to write a wrapper program that invokes DDS as well as the documentation tool (The DDS project does this very thing using [Dagon](https://github.com/vector-of-bool/dagon)).

**Reply by oraqlle** [reply]
Would it be possible to add it to a custom toolchain?
